nixos/tests/redmine.nix

```diff
@@ -10,18 +10,19 @@ let
   redmineTest = package: makeTest {
     machine =
       { config, pkgs, ... }:
-      { services.mysql.enable = true;
-        services.mysql.package = pkgs.mariadb;
-        services.mysql.ensureDatabases = [ "redmine" ];
-        services.mysql.ensureUsers = [
+      { services.postgresql.enable = true;
```

Comment:
Can you make redmine+postgresql a separate test? So both mysql and pg backends are tested?
In the end there will be 3 tests (3+mysql, 4+mysql, 4+pg)

Reply:
Sounds good